tests/imagebasedupgrade/internal/imagebasedupgradeinittools/imagebasedupgradeinittools.go

```diff
@@ -0,0 +1,47 @@
+package imagebasedupgradeinittools
```

Comment:
Let's add this directory to the gochecknoinits exclude rules in `.golangci.yml` file to avoid the lint failure.

Reply:
OK, done.